doc/source/examples/dem/packing-in-circle/packing-in-circle.rst

```diff
@@ -94,26 +90,23 @@ The ``insertion info`` subsection manages the insertion of particles.
       set insertion prn seed                             = 19
     end
 
-First, the ``insertion method`` is selected. There are two insertion methods (``uniform`` and ``non_uniform``) in Lethe-DEM. In ``uniform`` insertion, the particles are inserted uniformly (without randomness in their initial location), while in ``non_uniform``, particles are inserted randomly in the insertion box. ``inserted number of particles at each time step`` specifies the desired number of particles to be inserted at each insertion step.
-
-.. note ::
-	The meaning of randomness in the initial location of particles in a non-uniform insertion is structured randomness (Using pseudo-random number generator algorithms).
+First, the ``insertion method`` is selected. In ``volume`` insertion, the particles are inserted in an insertion box. The ``inserted number of particles at each time step`` specifies the desired number of particles to be inserted at each insertion step.
```

Comment:
we should rename the parameter ``inserted number of particles at each time step`` to either ``inserted number of particles at each time step`` or ``inserted number of particles at each insertion step``. 

We are not inserting at each time step.

Reply:
(FYI, this comment is not specific to this PR. I can start a issue regarding this. I just realized this during the review :) )